lib/spritely/sprockets/preprocessor.rb

```diff
@@ -32,6 +34,10 @@ def _call(input)
         end
       end
 
+      def process_directory_directive(value)
```

Comment:
This is a separate method definition rather than being done as part of the `GLOBAL_DIRECTIVES` constant because we don't want the `:directory` option to be merged into each image's options, it should be a separate configuration.